app/tests/challenges_tests/test_forms.py

```diff
@@ -78,3 +79,39 @@ def test_accept_challenge_request(
         f"There already is a challenge with short name: {type_1_challenge_request.short_name}"
         in str(form.errors)
     )
+
+
+@pytest.mark.django_db
+def test_budget_update_form(
+    client, challenge_reviewer, type_2_challenge_request

```

Comment:
Are `client` and `challenge_reviewer` required here?

Reply:
Neither are required here, thanks for catching this!